.spectral.yaml

```diff
@@ -0,0 +1,16 @@
+documentationUrl: https://github.com/italia/api-oas-checker-rules
+extends:
+  [
+    "https://github.com/italia/api-oas-checker-rules/releases/download/1.1/spectral-modi.yml",
+  ]
+rules:
+  servers-use-https: off
+  paths-status: off # Enable this once authorization server migration is completed and we can remove the /authorization-server prefix
+  paths-kebab-case: off # Disabled for compatibility with current endpoint naming
+  cache-responses-indeterminate-behavior: off # Disabled to avoid warnings on cache management
+  missing-ratelimit: off # Disabled to avoid warnings on rate limit headers
+  no-method-name-in-operationId: off # Disabled to avoid warnings on method names in operationIds
+  missing-retry-after: off
+  operation-description: off # Will be handled in the future
+  http-request-DELETE-no-body: off
+  allowed-number-format: off
```

Comment:
Note: will be handled and enabled in further tasks, see https://pagopa.atlassian.net/browse/PIN-6993